source/renderer/app/components/widgets/forms/PinCode.js

```diff
@@ -64,9 +66,33 @@ export default class PinCode extends Component<Props> {
         this.inputsRef[inputFocusKey]
       )
         this.inputsRef[inputFocusKey].focus();
+    } else if (
+      key === 0 &&
```

Comment:
`key === 0` can be removed here because it can only be zero

Reply:
Fixed